po/extra/pt.po

```diff
@@ -1,107 +1,118 @@
 #: data/com.github.gijsgoudzwaard.image-optimizer.appdata.xml.in:15
 msgid "Works offline, always ready where and whenever needed"
 msgstr ""
+"Funciona sem estar ligado à Internet, sempre pronto onde e quando precisar"
 
 #: data/com.github.gijsgoudzwaard.image-optimizer.appdata.xml.in:16
 msgid "A clean interface, optimize your images effortlessly"
-msgstr ""
+msgstr "Interface mais limpa, otimize as suas imagens sem esforço"
 
 #: data/com.github.gijsgoudzwaard.image-optimizer.appdata.xml.in:17
 msgid "Easily quit the application by pressing Ctrl + Q"
-msgstr ""
+msgstr "Saía da aplicação rapidamente pressionando em Ctrl + Q"
 
 #: data/com.github.gijsgoudzwaard.image-optimizer.appdata.xml.in:40
 msgid "Getting things ready for Juno, #27 should be fixed."
-msgstr ""
+msgstr "A preparar coisas para o Juno, #27 deve estar corrigido."
 
 #: data/com.github.gijsgoudzwaard.image-optimizer.appdata.xml.in:45
 msgid "Fixed unmet dependencies."
-msgstr ""
+msgstr "Dependências não satisfeitas corrigidas."
 
 #: data/com.github.gijsgoudzwaard.image-optimizer.appdata.xml.in:50
 #: data/com.github.gijsgoudzwaard.image-optimizer.appdata.xml.in:55
 msgid "Getting things ready for Juno."
-msgstr ""
+msgstr "A preparar coisas para o Juno."
 
 #: data/com.github.gijsgoudzwaard.image-optimizer.appdata.xml.in:60
 msgid "This release fixes the context menu in Files."
-msgstr ""
+msgstr "Este lançamento corrige o menu de contexto em Ficheiros."
 
 #: data/com.github.gijsgoudzwaard.image-optimizer.appdata.xml.in:65
 msgid ""
 "This release adds a context menu in Files for images that will allow easier "
 "image optimizations."
 msgstr ""
+"Este lançamento adiciona um menu de contexto em Ficheiros para imagens que "
+"vai tornar mais fácil a otimização das mesmas."
 
 #: data/com.github.gijsgoudzwaard.image-optimizer.appdata.xml.in:70
 msgid ""
 "This release changes the upload button to fit the Elementary OS HIG better."
 msgstr ""
+"Este lançamento altera o botão de envio para se integrar melhor na HIG do "
+"Elementary."
 
 #: data/com.github.gijsgoudzwaard.image-optimizer.appdata.xml.in:75
 msgid ""
 "This release changes the upload icon to fit the Elementary OS HIG better."
 msgstr ""
+"Este lançamento altera o ícone de envio para se integrar melhor na HIG do "
+"Elementary."
 
 #: data/com.github.gijsgoudzwaard.image-optimizer.appdata.xml.in:80
 msgid "This release changes the icon to fit the Elementary OS HIG better."
 msgstr ""
+"Este lançamento altera o ícone para se integrar melhor na HIG do Elementary."
 
 #: data/com.github.gijsgoudzwaard.image-optimizer.desktop.in:5
 msgid "Simple image optimzer"
-msgstr ""
+msgstr "Otimizador de imagem simples"
 
 #: data/com.github.gijsgoudzwaard.image-optimizer.desktop.in:8
 msgid "com.github.gijsgoudzwaard.image-optimizer"
-msgstr ""
+msgstr "com.github.gijsgoudzwaard.image-optimizer"
 
 #: data/com.github.gijsgoudzwaard.image-optimizer.desktop.in:12
 msgid "Image;Optimzer;"
-msgstr ""
+msgstr "Imagem;Otimizador;"
```

Comment:
It's fine if you translate this, but the original title should be in there as well since the name of the app shouldn't be translated.

Reply:
Done